e2e/api/steps.js

```diff
@@ -1016,6 +1018,25 @@ module.exports = {
       await updateAndAssertCaseFlag(value, partyFlags[index], caseId);
     }
   },
+
+  scheduleHearing: async (user, allocatedTrack) => {
+  console.log('Hearing Scheduled for case id ' + caseId);
+  await apiRequest.setupTokens(user);
+
+  eventName = 'HEARING_SCHEDULED';
+
+  caseData = await apiRequest.startEvent(eventName, caseId);
+  delete caseData['SearchCriteria'];
+
+  let scheduleData = data.HEARING_SCHEDULED(allocatedTrack);
+
+  for (let pageId of Object.keys(scheduleData.valid)) {
+    await assertValidData(scheduleData, pageId);
+  }
+
+  await assertSubmittedEvent('HEARING_READINESS', null, false);
```

Comment:
Tests are currently failing here, because the event end state is wrong. Once corrected, the tests should run as well.

Reply:
Fix has been merged